apps/pie-docs/src/accessibility/testing-for-accessibility.md

```diff
@@ -0,0 +1,135 @@
+---
+eleventyNavigation:
+    key: Testing for accessibility
+    parent: Accessibility
+    order: 3
+title: Testing for accessibility
+description: Find out more about how we test for accessibility and ensure ensure that our products meet accessibility standards.
+---
+
+## Why do we need to test for accessibility?
+
+At JET we understand that accessibility is crucial when it comes to designing and building products. We are aiming for our products and services to meet at least **level AA** of the **Web Content Accessibility Guidelines 2.1 (WCAG)**. To achieve this, it's essential that you consider accessibility at the start of a project and run accessibility tests regularly throughout the design process.
+
+We offer guidance on the different types of accessibility testing that should be used in your projects. We recommend a **mixture of automated, manual, and user testing  methods** to ensure that most accessibility issues can be detected.
+
+{% contentPageImage {
+    src:"../../../assets/img/accessibility/testing-for-accessibility/aa-wcag.svg",
+    mobileSrc:"../../../assets/img/accessibility/testing-for-accessibility/aa-wcag_narrow.svg",
+    caption: "Our products should meet the AA level of WCAG 2.1’s accessibility guidelines."
+} %}
+
+---
+
+## Automated testing
+
+Automated testing tools can audit your product against the WCAG to ensure that you're following best practices that can be detected.
+
+We estimate that **automated testing can detect a great amount of common WCAG issues**. Automated testing also gives your team more time to focus on the more complex accessibility tests that need to be completed manually.
```

Comment:
Same here – I would right this as:

```suggestion
We estimate that **automated testing can detect a large number of common WCAG issues**. Automated testing also gives your team more time to focus on the more complex accessibility tests that need to be completed manually.
```

Reply:
Agree with all suggestions, let's amend it :D